src/probnum/problems/zoo/filtsmooth/_filtsmooth_problems.py

```diff
@@ -0,0 +1,470 @@
+    initcov = 3.0 * np.eye(1)
+    initrv = randvars.Normal(initmean, initcov)
+    dynamics_model = statespace.SDE(dimension=1, driftfun=f, dispmatfun=l, jacobfun=df)
+    measurement_model = statespace.DiscreteLTIGaussian(
+        state_trans_mat=np.eye(1),
+        shift_vec=np.zeros(1),
+        proc_noise_cov_mat=measurement_variance * np.eye(1),
+    )
+
+    statespace_components = dict(
+        dynamics_model=dynamics_model,
+        measurement_model=measurement_model,
+        initrv=initrv,
+    )
+    return statespace_components
+
+
+def logistic_ode(
+    y0: Optional[Union[np.ndarray, FloatArgType]] = None,
+    solver_initrv: Optional[randvars.RandomVariable] = None,
+    timespan: Optional[Tuple[float, float]] = None,
+    params: Optional[Tuple[float, float]] = None,
+    evlvar: Optional[Union[np.ndarray, FloatArgType]] = None,
+    ek0_or_ek1: Optional[int] = None,
+):
+    r"""Filtering/smoothing setup for a probabilistic ODE solver based on the logistic ODE.
+
+    This state space model puts an integrated Brownian motion prior on the dynamics
+    and constructs the ODE likelihood based on the vector field defining the
+    logistic ODE.
+
+    Parameters
+    ----------
+    y0
+        Initial conditions of the Initial Value Problem
+    solver_initrv
+        Initial random variable of the probabilistic ODE solver
+    timespan
+        Time span of the problem
+    params
+        Parameters for the logistic ODE
+    evlvar
+        See :py:class:`probnum.diffeq.GaussianIVPFilter`
+    ek0_or_ek1
+        See :py:class:`probnum.diffeq.GaussianIVPFilter`
+
+    Returns
+    -------
+    logistic_ivp
+        The initial value problem based on the logistic ODE.
+    statespace_components
+        Dictionary containing
+        - dynamics model
+        - measurement model
+        - initial random variable
+
+    See Also
+    --------
+    :py:class:`probnum.diffeq.GaussianIVPFilter`
+
+    """
+
+    if timespan is None:
+        timespan = (0.0, 2.0)
+
+    if y0 is None:
+        y0 = [0.1]
+    y0 = np.array(y0)
+
+    if params is None:
+        params = (6.0, 1.0)
+
+    if evlvar is None:
+        evlvar = np.zeros((1, 1))
+
+    if ek0_or_ek1 is None:
+        ek0_or_ek1 = 1
+
+    logistic_ivp = diffeq.logistic(
+        timespan=timespan, initrv=randvars.Constant(y0), params=params
+    )
+    dynamics_model = statespace.IBM(ordint=3, spatialdim=1)
+    measurement_model = filtsmooth.DiscreteEKFComponent.from_ode(
+        logistic_ivp, prior=dynamics_model, evlvar=evlvar, ek0_or_ek1=ek0_or_ek1
+    )
+
+    if solver_initrv is None:
+        initmean = np.array([0.1, 0, 0.0, 0.0])
+        initcov = np.diag([0.0, 1.0, 1.0, 1.0])
+        solver_initrv = randvars.Normal(initmean, initcov)
+
+    statespace_components = dict(
+        dynamics_model=dynamics_model,
+        measurement_model=measurement_model,
+        initrv=solver_initrv,
+    )
+    return logistic_ivp, statespace_components
```

Comment:
I think the return value should be a regression problem with a grid and zero-data (this is a filtsmooth zoo :)), and the logistic ODE defines the measurement that is returned as part of the state space components. what do you think?